docs/sources/flow/getting-started/migrating-from-prometheus.md

```diff
@@ -0,0 +1,183 @@
+The contents of the new flow configuration:
+
+```river
+prometheus.scrape "prometheus" {
+	targets = [{
+		__address__ = "localhost:12345",
+	}]
+	forward_to     = [prometheus.remote_write.default.receiver]
+	job_name       = "prometheus"
+	scrape_timeout = "45s"
+}
+
+prometheus.remote_write "default" {
+	endpoint {
+		name = "grafana-cloud"
+		url  = "https://prometheus-us-central1.grafana.net/api/prom/push"
+
+		basic_auth {
+			username = "USERNAME"
+			password = "PASSWORD"
+		}
+
+		queue_config {
+			capacity             = 2500
+			max_shards           = 200
+			max_samples_per_send = 500
+		}
+
+		metadata_config {
+			max_samples_per_send = 500
+		}
+	}
+}
+```
+
+### Debugging
+
+1. If Prometheus configuration is provided that cannot be converted,
+   diagnostic information is printed to `stderr`. You can bypass
+   any non-critical issues and output the flow configuration using best
+   effort conversion by including the `--bypass-errors` flag.
+   
+    > Be aware that the behavior may not match when bypassing errors.
+
+    ```bash
+    grafana-agent convert --format=prometheus --bypass-errors --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+    ```
+
+2. You can also output a diagnostic report by including the `--report` flag.
+
+    ```bash
+    grafana-agent convert --format=prometheus --report=OUTPUT_REPORT_PATH --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+    ```
+
+    1. Replace `OUTPUT_REPORT_PATH` with the full path to output the report to.
+
+    Using the example Prometheus configuration from above outputs a diagnostic
+    report:
+
+    ```
+    (Info) Converted scrape_configs job_name "prometheus" into...
+      A prometheus.scrape.prometheus component
+    (Info) Converted 1 remote_write[s] "grafana-cloud" into...
+      A prometheus.remote_write.default component
+    ```
+
+## Run a Prometheus configuration
+
+If you’re not ready to switch over to flow configuration, you can also run
+the Prometheus configuration without having to save it as a flow config.
+This allows you to try flow mode without having to modify your existing
+Prometheus configuration infrastructure.
+
+> In this task, we will use the [run][] CLI command to run the Agent in flow
+> mode using a Prometheus configuration.
+
+1. [Start the agent][] in flow mode and include the command line flag
+   `--config.format=prometheus`. Your configuration file should be a valid
+   Prometheus configuration rather than a flow mode configuration
+
+
+### Debugging
+
+1. The convert CLI command [debugging][] instructions can be followed to
+   generate a diagnostic report.
+
+2. See the Grafana Agent [Flow Debugging][] for debugging a running Grafana
+   Agent in flow mode.
+
+3. If the Prometheus configuration provided cannot be converted,
+   diagnostic information is printed to `stderr`. You can bypass
+   any non-critical issues and start the Agent by including the
+   `--config.bypass-conversion-errors` flag in addition to
+   `--config.format=prometheus`.
```

Comment:
This is a little confusing. We state that you're not converting to flow config and just using your existing Prometheus config... but this step says the Prometheus config can't be converted so info is sent to stderr.  Is it converting on the fly behind the scenes? do we need to make this clearer in the section above the debugging?


Also... do you print to stderr? Write to? send to?

Reply:
I updated to say `sent to` instead of `printed to`

We are converting the config to flow using the same backend functionality and then running it up as a flow config file instead of writing it to an output. What do you think we could add for clarity around this?